package.json

```diff
@@ -4,11 +4,8 @@
   "description": "Health check app with Slack Notification",
   "main": "index.js",
   "dependencies": {
-    "@slack/client": "^4.1.0",
-    "fs": "0.0.1-security",
-    "http": "0.0.0",
-    "https": "^1.0.0",
-    "node-emoji": "^1.8.1",
+    "dotenv": "^16.0.0",
```

Comment:
you can add dotenv as devDependency. In my case I'm using systemd to load the env, so this doesn't need to be installed (npm install --production)